source/actions3/actions/AMDetectorSetDarkCurrentValueActionInfo.cpp

```diff
@@ -0,0 +1,32 @@
+#include "AMDetectorSetDarkCurrentValueActionInfo.h"
+
+AMDetectorSetDarkCurrentValueActionInfo::AMDetectorSetDarkCurrentValueActionInfo(double newValue, const AMDetectorInfo &detectorInfo, QObject *parent) :
+    AMActionInfo3(QString(), QString(), QString(), parent)
```

Comment:
Because the name and description are the same as the typeDescription(), I opted to set them in the subclass constructor using setShortDescription() and setLongDescription(). Should this be done another way?